pkgs/applications/misc/fitnesstrax/default.nix

```diff
@@ -0,0 +1,44 @@
+{ pkgs,
+  fetchFromGitHub ? pkgs.fetchFromGitHub,
+  stdenv ? pkgs.stdenv,
+  rustPlatform ? pkgs.rustPlatform,
+  wrapGAppsHook ? pkgs.wrapGAppsHook,
+}:
+rustPlatform.buildRustPackage rec {
+  name = "fitnesstrax-${version}";
+  version = "0.1.0";
+
+  src = fetchFromGitHub {
+    owner = "luminescent-dreams";
+    repo = "fitnesstrax";
+    rev = "516a9e48978ac0f90bbbeade34d626211ccb78a3";
+    sha256 = "1k6zhnbs0ggx7q0ig2abcnzprsgrychlpvsh6d36dw6mr8zpfkp7";
+  };
+
+  nativeBuildInputs = [
+    pkgs.glib
+    pkgs.atk
+    pkgs.gnome2.pango
+    pkgs.cairo
+    pkgs.glib
+    pkgs.gdk_pixbuf
+    pkgs.gtkd
+    pkgs.gtk3
+    wrapGAppsHook
+    ];
```

Comment:
please reference these formally (without pkgs). All of these except `wrapGAppsHook` should be `buildInputs` also.

Reply:
Split. only needed wrapGAppsHook in nativeBuildInputs, and the rest in work in buildInputs.